x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/notify_user_option.tsx

```diff
@@ -150,6 +152,93 @@ export const NotifyUserOption = React.memo(
       [protection]
     );
 
+    const customNotificationComponent = useMemo(() => {
+      if (!userNotificationSelected) {
+        return null;
+      }
+
+      if (CustomNotificationUpsellingComponent) {
+        return <CustomNotificationUpsellingComponent />;
+      }
+
+      if (!isEditMode) {
+        return (
+          <>
+            <EuiSpacer size="m" />
+            <EuiText size="s">
+              <h4>{NOTIFICATION_MESSAGE_LABEL}</h4>
+            </EuiText>
+            <EuiSpacer size="xs" />
+            <>{userNotificationMessage || getEmptyValue()}</>
```

Comment:
Q: Does `emptyValue()` (-) ever get rendered? from the types looks like `userNotificationMessage` is always present.

Reply:
I believe that userNotificationMessage can be an empty string, which works fine when displaying it in the input box. However, in view-only mode, we want to indicate graphically that there is no notification message by displaying -.

Reply:
Yeah, but I don't see the `-` in the view only mode. I still see an empty textarea. 😕  I'll test again.

Reply:
User with full access:
![Screenshot 2024-10-01 at 10 45 59](https://github.com/user-attachments/assets/d89cbdb5-b2a1-4ec5-af33-c14c18136609)

User with read only access:
![Screenshot 2024-10-01 at 10 49 17](https://github.com/user-attachments/assets/bfba0635-c974-4b33-b43a-8a4735167c77)

Seems like empty string is being replaced with the default message here - https://github.com/elastic/kibana/blob/847285ba7191aa6d26fb3dccc06748e1c4a202b1/x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/middleware/policy_settings_middleware.ts#L44

I didn’t feel comfortable enough removing the `|| getEmptyValue()`, so I decided to leave it in 😉

Reply:
Yeah, this is updating it to the `Elastic Security {action} {filename} ` text. Which I think should be updated to `Elastic Endpoint {action} {filename} ` if at all that is the behavior we need. Maybe @paul-tavares @dasansol92 know something about this logic?

Reply:
That sounds good to me.